Component Plug-in (CP) Examples/backgroundColor/appian-component-plugin.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<appian-component-plugin name="Background Color" key="com.mycorp.background.color">
+  <plugin-info>
+    <description>Provides a Background Color component to demo multi-select dropdowns</description>
+    <vendor name="Appian" url="http://www.appian.com" />
+    <support supported="true" email="support@appian.com" url="https://community.appian.com"/>
+    <version>1.0</version>
+  </plugin-info>
+
+  <component rule-name="backgroundColor" version="1.0.0">
+    <sdk-version>2.0.0</sdk-version>
+    <parameter>
+      <name>colors</name>
+      <category>input-only</category>
+      <type>
+        <enum multiple="true">
+          <choice>RED</choice>
+          <choice>BLUE</choice>
+        </enum>
+      </type>
+    </parameter>
+    <html-entry-point>index.html</html-entry-point>
+    <supported-user-agents>chrome firefox ie11 edge safari mobile</supported-user-agents>
```

Comment:
Can we confirm that this works for mobile?

Reply:
Yep it does